.drone.yml

```diff
@@ -29,7 +29,10 @@ steps:
       - apt-get update
       - apt-get -y install protobuf-compiler libprotobuf-dev
       - rustup component add clippy
-      - cargo clippy --workspace --tests --all-targets --all-features -- -D warnings -D deprecated -D clippy::perf -D clippy::complexity -D clippy::dbg_macro
+      - cargo clippy --workspace --tests --all-targets --all-features -- \
+          -D warnings -D deprecated -D clippy::perf -D clippy::complexity \
+          -D clippy::dbg_macro -D clippy::inefficient_to_string \
+          -D clippy::items-after-statements -D clippy::implicit_clone
```

Comment:
Here are the lints im adding, starting from clippy::inefficient_to_string. You can look them up [here](https://rust-lang.github.io/rust-clippy/master/index.html).